docs/operators/csvParserOperator.md

```diff
@@ -0,0 +1,80 @@
+Csv Parser Operator
+=============
+
+## Operator Objective
+This operator is designed to parse delimited records and construct a map or concrete java class also known as ["POJO"](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) out of it. User need to provide the schema to describe the delimited data. Based on schema definition the operator will parse the incoming record to object map and POJO.  User can also provide constraints if any, in the schema. The supported constraints are listed at http://super-csv.github.io/super-csv/cell_processors.html. The incoming record will be validated against those constraints. Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message.
+
```

Comment:
Not all processors are currently supported. I recommend to include a table listing down supported processors.